qa/rpc-tests/bigblocks.py

```diff
@@ -198,8 +214,22 @@ def run_test(self):
         # Time starts a little before fork activation time:
         set_node_times(self.nodes, FORK_TIME - 100)
 
-        # No supermajority
-        self.TestMineBig(False)
+        # No supermajority, vote2mb=1
+        self.TestMineBig(False, True)
+
+        # Create a block after the expiration date. This will be rejected 
+        # by the other nodes for being more than 2 hours in the future,
```

Comment:
So if it's asserted that the block will be rejected for being more than 2 hours in the future, doesn't that mean that the test will fail once the date arrives when it's no longer more than 2 hours in the future?